res/Sectors/Distant Fringe/Abyss.txt

```diff
@@ -1,11 +1,11 @@
 Hex  Name                 UWP       Remarks              {Ix} (Ex) [Cx] N B Z PBG W  A  Stellar     
 ---- -------------------- --------- -------------------- ---- ---- ---- - - - --- -- -- ------------
-0102 Jorutha              ?311???-? Ic Tz                               - - - ?11 8  Nx M4 V        
-0901 Masikaro             ?6A4???-? Fl Tz                               - - - ?22 10 Nx M3 V M7 V   
-1103 Gerund               ?512???-? Ic Tz                               - - - ?30 7  Nx M6 V        
-1706 Erim Arut            ?650???-? De He Po                            - - - ?30 11 Nx M7 III M2 V 
-2803 Boroyen              ?421???-? He Po Sa                            - - - ?01 6  Nx M8 V        
-2903 Darga                ?000???-? As Va                               - - - ?11 7  Nx M7 V M8 V    
-1211 Farisil              ?686???-? Ga Tz                               - - - ?10 7  Nx K1 V        
-1129 Allathur             ?6A3???-? Fl Tz                               - - - ?21 5  Nx M0 V        
-2336 Hamarah              ?432???-? Po Tz                               - - - ?10 8  Nx M2 V        
\ No newline at end of file
+0102 Jorutha              ?311???-? Ic Tz                               - - - ?11 8  NaXX M4 V        
```

Comment:
With the header untouched this is invalid. Make sure to use travellermap.com/tools/lintsec to validate any updated data.